lib/inputs/file-pattern/static.ts

```diff
@@ -48,14 +48,15 @@ export function getMatchingFiles(
       if (actionName !== "find-files-by-pattern") {
         continue;
       }
-      if (!(typeof extractedLayers[filePath][actionName] === "string")) {
-        throw new Error("expected string");
+
+      if (!Buffer.isBuffer(extractedLayers[filePath][actionName])) {
+        throw new Error("expected a buffer");
       }
 
       manifestFiles.push({
         name: path.basename(filePath),
         path: path.dirname(filePath),
-        contents: extractedLayers[filePath][actionName] as string,
+        contents: extractedLayers[filePath][actionName] as Buffer,
```

Comment:
why do you need the `as Buffer`?

Reply:
The values of extractedLayers are either string or Buffer - manifestFiles needs a Buffer for the contents field (see my change in types.ts) - as before, we need to be explicit what we are assigning here. This is the same as before - before you had to say it is a string.